app/models/alchemy/translations.rb

```diff
@@ -2,7 +2,7 @@ module Alchemy
   module Translations
     # we want to prepend all translated data bound four our translation engine
     # with this prefix for orgainizational/namespace reasons
-    TRANSLATION_PREFIX = 'cms'
+    TRANSLATION_PREFIX = ENV['TRANSLATION_PREFIX']
```

Comment:
TRANSLATION_PREFIX = ENV['TRANSLATION_PREFIX'] || 'cms' ?  would like to have a sane default in the event of missing env var